src/lib/components/billing/BillingPage.svelte

```
@@ -95,144 +89,51 @@
+            window.location.href = "/auth";
+          } else if (isUnlimited && isLoggedIn && !isUserSubscribed) {
+            $createCheckoutSession.mutate({ plan: "unlimited" });
+          }
+        }}
+        isPopular={isUnlimited && !isUserSubscribed ? plan.isPopular : false}
+        isPrimary={plan.isPrimary}
+        customBadgeText={isUnlimited && isUserSubscribed ? "Current Plan" : null}
+        customBadgeIcon={isUnlimited && isUserSubscribed ? Star : null}
+        isButtonDisabled={isDisabled}
+      />
+    {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)